node/db/handcraft_data.sql

```diff
@@ -1,22 +1,31 @@
-INSERT INTO users (username, name, bio, password, type) VALUES ('SergeyBrin','Sergey Brin','CEO at Google.','password', 'admin');
+INSERT INTO users (username, name, bio, password, type) VALUES ('Docker','Docker','Docker is the leader in the containerization market, combining an enterprise-grade container platform with world-class services.','password', 'company');
 INSERT INTO users (username, name, bio, password, type) VALUES ('matthew-jay-wong','Matthew Wong','3rd Year CS Specialist.','password', 'student');
 INSERT INTO users (username, name, bio, password, type) VALUES ('3370sohail','Sohail Hameed','3rd Year CS Specialist.','password', 'student');
-INSERT INTO users (username, name, bio, password, type) VALUES ('TensorFlow','TensorFlow','Open source software library.','password', 'company');
 INSERT INTO users (username, name, bio, password, type) VALUES ('Clarifai','Clarifai','Image processing library.','password', 'company');
+INSERT INTO users (username, name, bio, password, type) VALUES ('TensorFlow','TensorFlow','Open source software library.','password', 'company');
+INSERT INTO users (username, name, bio, password, type) VALUES ('Maven','Maven','Apache Maven is a software project management and comprehension tool.','password', 'company');
+INSERT INTO users (username, name, bio, password, type) VALUES ('Firebase','Firebase','Google''s mobile platform that helps you quickly develop high-quality apps and grow your business.','password', 'company');
+INSERT INTO users (username, name, bio, password, type) VALUES ('Flutter','Flutter','Flutter is Google’s mobile app SDK for crafting high-quality native interfaces on iOS and Android in record time.','password', 'company');
 

```

Comment:
Could you create fake users instead of using projects as users?

Reply:
Like making up names of people who study at UofT, describe some of their hobbies and talk about courses they like.